tests/integrationtests/custom_cost_aggregation_testfiles/by_thread.txt

```diff
@@ -0,0 +1,15 @@
+
+	cpp-threadnames
+		_dl_lookup_symbol_x
+			_dl_relocate_object
+				dl_main
+					_dl_sysdep_start
+						_dl_start
+							_start
+		do_lookup_x
```

Comment:
dito above

also, generally, it would be much better to use a different cost, such as off-CPU time or scheduler switches instead of cycles for this comparison. I would then also expect the other threads to be actually shown here, which currently does not seem to be the case, making me doubt that the aggregation actually works?